servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.h

```diff
@@ -184,6 +184,7 @@ class RenderForwardClustered : public RendererSceneRenderRD {
 	};
 
 	enum {
+		INSTANCE_DATA_FLAGS_NON_UNIFORM_SCALE = 1 << 5,
```

Comment:
Just in case you wonder about this change, I will use the upper 8 bits for something else soon.